### ISSUE TITLE
[Feature Request] Install via homebrew and other package managers

### ISSUE THREAD

**Issue by Gidgidonihah** [gruntwork-io/git-xargs#12]
Title says it all.

**Comment by bdarfler** [comment]
I wanted to expand on my +1 here. I tried installing via the `go get` option and ran into `cannot find package "github.com/google/go-github/v32/github"` which led me to this issue https://github.com/google/go-github/issues/1641 which didn't actually resolve the issue for me.

Was finally able to run `go get` from inside the `git-xargs` folder that was pulled when I ran `go get github.com/gruntwork-io/git-xargs` but this was after a number of dead ends and I was only able to get there because I have enough go exposure to be dangerous.

So I do have git-xargs installed but now I'm leaning towards experimenting with https://github.com/lindell/multi-gitter first since the install experience gives me more confidence in the project.

I say all this not to be a jerk but because I want to give a specific, actionable user experience report in hopes that it can help this project.
